### PR TITLE
Transport.cpp: provide transport congestion client with RTCP Receiver…

### PR DESCRIPTION
**Pull request by jmillan**
… Reports

### CODE REVIEW

File: worker/src/RTC/Transport.cpp [hunk 1]
```diff
@@ -1914,9 +1916,13 @@ namespace RTC
 						continue;
 					}
 
+					rtt = consumer->GetRtt();
```
Comment (ibc) [inline worker/src/RTC/Transport.cpp]: What about the commented block above? Shouldn't we also pass RR when it belongs to the probator?

Comment (jmillan) [reply worker/src/RTC/Transport.cpp]: correct.

Comment (jmillan) [reply worker/src/RTC/Transport.cpp]: In fact the API in `RtpTransportControllerSend` expects a list of reports, not just the one of the probator (as suggested in the commented block above) and the libwebrtc code does not distinguish between probator or other streams in order to provide with the RR's. I'll consider the probator consumer `rtt` as well.

Comment (jmillan) [reply worker/src/RTC/Transport.cpp]: Indeed there is no consumer for probation.

Comment (jmillan) [reply worker/src/RTC/Transport.cpp]: comment block removed.

Comment (ibc) [reply worker/src/RTC/Transport.cpp]: There is no consumer for probator, right, but we should be able to read RR from probator and pass it to the bwe, am I wrong?

Comment (jmillan) [reply worker/src/RTC/Transport.cpp]: oh, that's right. I've just ignore probator RR...

Comment (jmillan) [reply worker/src/RTC/Transport.cpp]: > There is no consumer for probator, right, but we should be able to read RR from probator and pass it to the bwe, am I wrong?

It's done already.

Comment (ibc) [reply worker/src/RTC/Transport.cpp]: Where?

Comment (jmillan) [reply worker/src/RTC/Transport.cpp]: after the loop: https://github.com/versatica/mediasoup/pull/464/files#diff-d9a2ff41c9bac4d9444a0ea711677398R1906

Comment (ibc) [reply worker/src/RTC/Transport.cpp]: If the RR belongs to the probator we do `continue` so `rtt` is 0., It is ok?

Comment (jmillan) [reply worker/src/RTC/Transport.cpp]: the whole RTCP packet is provided after iterating.

Comment (jmillan) [reply worker/src/RTC/Transport.cpp]: We take the RTT from one of the Consumers, probator does no have any so it's not considered for the RTT.

Comment (jmillan) [reply worker/src/RTC/Transport.cpp]: > If the RR belongs to the probator we do continue so rtt is 0., It is ok?

It's a continue, not a break, we keep iterating the Receiver Reports.

**Pull request merged**